src/relion/zocalo/tomo_align.py

```diff
@@ -231,6 +235,12 @@ def tilt(file_list):
         else:
             self.log.warning(f"{self.central_slice_location} hasn't been written yet")
 
+        m = Path(self.stack_movie_location)
+        if m.is_file():
```

Comment:
Is this checking for the movie that the images service outputs?

Reply:
Yes, so that an ISPyB insert isn't attempted unless the file exists

Reply:
I'm confused about the order. Doesn't the message get sent to the images service after this?

Reply:
I've gotten confused somewhere because yes. you're right. The main point of that code is to change the permissions of the files, but I thought there was some check in there or that logs like [this](https://graylog2.diamond.ac.uk/messages/graylog_35616/612c47c1-8d52-11ed-bee7-1866dafae904) were useful for some reason.

Reply:
I think it will never be there so you always get the warning. Is that right? I think we can cut it

Reply:
Yes, I think so, I'll remove it. And the images service does it's own insert so no check for file existing is needed there (doing the insert in the images service was the fix for that problem, I remember now)

Reply:
Cool, thanks